student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(id, age, name: 'Unknown', parent_permission: true, classroom: nil)
+    super(id, age, name: name, parent_permission: parent_permission)
```

Comment:
- [ **OPTIONAL** ] **RECOMENDED:** As mentioned by the previous reviewer, I think it would be better if you implement the change he recommended  in the `initialize` method while creating both `Student` and `Teacher` as the `id` will be automatically generated with the `Random` method in the `Person` call. Both classes should look similar to this:
```ruby
# Student class
class Student < Person 
  def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
  end
...
end

# Teacher Class
class Teacher < Person
  def initialize(age, specialization, name = "Unknown", parent_permission = true)
    super(age, name, parent_permission)
    @specialization = specialization
  end
  ...
```